src/Files.App/ViewModels/Properties/HashesViewModel.cs

```diff
@@ -103,6 +103,11 @@ private void ToggleIsEnabled(string? algorithm)
 					{
 						// not an error
 					}
+					catch (IOException)
+					{
+						// File is currently open
+						hashInfoItem.HashValue = "CalculationErrorFileIsOpen".GetLocalizedResource();
+					}
```

Comment:
This would also catch the exception if the file had already been deleted.

Reply:
I think that's an edge case, but there are ways to handle it if users start opening issues regarding this.